### PR TITLE
PEP8 tests and enable flake8 run on tests in CI

### PR DESCRIPTION
**Pull request by etingof**
@sydneyli this is a highly mundane patch. Just in case you are not sufficiently bored. ;)

### CODE REVIEW

File: tests/unit/test_lexer.py [hunk 1]
```diff
@@ -68,24 +76,32 @@ def testExpressionTags(self):
...
+                '    ', ('if', ' ', 'a == 1'), '\n    ', 'if', '\n    '
+            ]
+        )
 
     def testHashEscapes(self):
-        text = "favorite_color \#000000"
+        text = "favorite_color \\#000000"
         tokens = self.lexer.tokenize(text)
-        self.assertEqual(tokens, [('favorite_color', ' ', '\#000000')])
+        self.assertEqual(tokens, [('favorite_color', ' ', '\\#000000')])
```
Comment (sydneyli) [inline tests/unit/test_lexer.py]: good catch on the escapes!

**Comment by etingof** [comment]
> Awesome, thanks! I'll rebase my other PRs so that they pass the linter.

That's what I have been hesitant to suggest! ;-)

**Comment by sydneyli** [comment]
Feel free to merge this in whenever (I don't think I can)

**Comment by etingof** [comment]
Right, sorry! I somehow thought you'd rebase first. But that's nonsense. Merged!

**Pull request merged**